pkg/security/probe/stateful_probe_excluder.go

```diff
@@ -0,0 +1,73 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+package probe
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/DataDog/ebpf-manager/tracefs"
+	"github.com/cilium/ebpf"
+)
+
+type availableFunctionsBasedExcluder struct {
```

Comment:
Ok, you can ignore my comment about the other comments, but here, I legit would like a comment since I think I know what you're saying here, but not sure